src/inc/gcdump.h

```diff
@@ -12,6 +12,12 @@
  * to the standard code-manager spec.
  */
 
+// ******************************************************************************
```

Comment:
What are the values uses by SOS from this file? I do not think that there is anything sensitive in this file wrt. the contract versioning.

Reply:
GCInfoToken is used by the gcdump library which is used by various SOS commands to dump the gcinfo.  The gcinfo hasn't changed since 1.0 and this is one of the 3 things I have found that doesn't go through the DAC.  The other two are the stresslog (dumplog) support and the special thread dump support (uses tls.sh/predeftlsslot.h).

Reply:
I was commenting on this specific file (gcdump.h). GCInfoToken is not defined in this file. 

The comment on the other GCInfo decoding related files is fine.

Reply:
> gcinfo hasn't changed since 1.0

It did change a few times since .NET Core 1.0, e.g. https://github.com/dotnet/coreclr/pull/6325/commits/f915aebaf5db0b829f062dc9940e23bb5c38d575

Reply:
Yes, I was getting the h files mixed up.  gcdump.h doesn't include any runtime specific stuff.